src/helpers.rs

```diff
@@ -846,6 +846,17 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
         self.set_last_error(self.io_error_to_errnum(err)?)
     }
 
+    /// Sets the last OS error using a `std::io::ErrorKind` and writes -1 to dest place.
+    fn set_io_error_and_return_neg_one(
```

Comment:
Should be `set_last_error_and...`. It's not setting an io error, after all.

Maybe just "and return" is enough, leaving `-1` as the usual convention?

So what about `set_last_error_and_return`.